apps/mark/backend/src/app.test.ts

```diff
@@ -31,31 +47,29 @@ test('uses machine config from env', async () => {
     VX_SCREEN_ORIENTATION: 'landscape',
   };
 
-  const { apiClient, server } = createApp();
+  ({ apiClient, server } = createApp());
```

Comment:
To your comment regarding making it possible to use an `afterEach` for server closing, the main alternative I can think of is making all the variables global but defining them in a `beforeEach`, e.g.

```ts
let apiClient: grout.Client<Api>;
let mockAuth: InsertedSmartCardAuthApi;
let mockUsb: MockUsb;
let server: Server;

beforeEach(() => {
  ({ apiClient, mockAuth, mockUsb, server } = createApp());
});

afterEach(() => {
  server.close();
});
```

This should make this concern you mention impossible:

> This means you risk forgetting to destructure eg. apiMock in a test, and then you're at risk of using an old apiMock from another test.

Reply:
[361e658](https://github.com/votingworks/vxsuite/pull/3223/commits/361e6588a0ef7f36b2b04ecc48d125a8dddb1275)

Great point. Had it in my head that we should only be destructuring exactly the fields we need, which in retrospect is a constraint that gains us nothing.